### DISCUSSION TITLE
Add option to mute dons or kats in osu!taiko editor

### DISCUSSION THREAD

**Discussion by pigeonskew** [ppy/osu#30475] [Ideas]
I think it would really help to map properly to the rhythm of the song. Maybe can also be implemented when watching a replay outside the editor.

**Comment by Tenexxt** [comment]
Currently, you can just set your effect volume to 0 and that should do it.

In the future it's planned to seperate the effect volume from hitsound volume, so you can mute hs but still have SFX, but for now thats not available so just mute your SFX and wait until it gets done

**Reply by pigeonskew** [reply]
No I mean, I want to specifically mute only either the dons or the kats. Not both of them at once. 
It's to check whether I mapped the dons or kats well with the music.
